src/lib/utils.sh

```diff
@@ -116,3 +116,41 @@ function send-log() {
 
   return 0
 }
+
+function sort-logs() {
+  set -euo pipefail
+
+  local CAUR_CACHE=="/tmp/chaotic/"
+
+  function cleanlogs
```

Comment:
Why should it be available outside of the sort-logs function? We already have a clean-logs function declared somewhere else - I've renamed it to distinguish it better.

Reply:
Because function declarations are always global. So instead of declaring a function like this:
```bash
function sort-logs
{
#content
}
```
You should declare it like this: 
```bash
function sort-logs()
(
#content
)
```
to avoid the function declaration from leaking into the global scope.

Reply:
@dr460nf1r3 this is not fixed yet.